app/src/componentLibrary/autocompletes/common/autocompleteOption/autocompleteOption.scss

```diff
@@ -99,10 +74,38 @@
   display: flex;
   justify-content: space-between;
   align-items: center;
+}
+
+.item, .new-item {
   min-height: 36px;
   padding: 4px 12px;
-  box-sizing: border-box;
   cursor: pointer;
+  box-sizing: border-box;
+
+  &.active:hover {
+    background-color: $COLOR--bg-100;
```

Comment:
as I can see we want bg-color = bg100 in two ablosutly opposite cases:
`&.active:hover` and `&.active:not(:hover)` Am I correct?

Reply:
Didn't catch this, thanks.